Multisig/UI/SwiftUI/Onboarding/LaunchView.swift

```diff
@@ -39,7 +39,7 @@ struct LaunchView: View {
     @State
     var showTerms = false
 
-    var onStart: () -> Void = { }
+    var onStart: () -> Void
```

Comment:
maybe this can be reverted and we don't need to add empty `{}` below?

Reply:
reverted